.github/workflows/openapi_update.yaml

```diff
@@ -0,0 +1,29 @@
+name: Dispatch OpenAPI File Change event
+on:
+  push:
+    branches: 
+      - main
+    paths:
+      - "openapi/*.yaml"
+
+jobs:
+  dispatch:
+    env:
+      APP_SERVICES_CI_TOKEN: ${{ secrets.GH_CI_TOKEN }}
+    strategy:
+      matrix:
+        repo:
+          - "redhat-developer/app-services-sdk-go"
+          - "redhat-developer/app-services-sdk-js"
+          - "redhat-developer/app-services-sdk-java"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        
+      - name: Sandbox OpenAPI changed
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ env.APP_SERVICES_CI_TOKEN }}
+          repository: ${{ matrix.repo }}
+          event-type: openapi-spec-change
+          client-payload: '{ "id": "openbridge-mgmt/v1", "download_url":"https://raw.githubusercontent.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/main/openapi/openapi.yaml"}'
```

Comment:
Let's use `rhose` instead of openbridge

Reply:
+100. We can also rename file to rhose sdk